language-support/ts/codegen/tests/ts/daml-json-types/src/index.ts

```diff
@@ -112,6 +112,26 @@ export const Decimal: Serializable<Decimal> = {
   decoder: jtv.string,
 }
 
+/**
+ * The counterpart of DAML's `Numeric` type. We represent `Numeric`s as string
+ * in order to avoid a loss of precision. The string must match the regular
+ * expression `-?(?:0|[1-9]\d*)(?:\.\d+)?(?:[eE][+-]?\d+)?`.
+ */
+export type Numeric = string;
+
+/**
+ * Companion function of the `Numeric` type.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const Numeric = function (_: number): Serializable<Numeric> {
+  // To-do : Write a smarter decoder to replace 'jtv.string' that
```

Comment:
Can you please make it `//TODO(SF, #xyz): ...`, where `#xyz` is the number of the issue for this. I prefer to be consistent on how to leave todos. That makes it easier to grep for todos.

Reply:
Ok.

Reply:
✔️